Model/Media.php

```diff
@@ -630,7 +629,7 @@ public function getCategory()
     /**
      * @param CategoryInterface $category|null
      */
-    public function setCategory(CategoryInterface $category = null)
+    public function setCategory($category = null)
```

Comment:
@Trip09 : it is a BC-break : a class that extends this class and overrides setCategory will have to change its signature if you remove the type hinting.